modules/special-rules/pages/air-transport-mission.adoc

```diff
@@ -27,20 +34,27 @@ NOTE: If a {transport-x} Flyer unit is destroyed while airborne then any embarke
 
 . Make ranged attacks with the Flyer units, just like you would with ground units in the Shooting phase -- use the Firepower values of the Flyer units and any super-heavy weapons that they have.
 But note:
-* You can draw a line of fire only where an enemy unit lies within range *and* inside the forward 90° arc of your Flyer unit.
-Ignore any intervening terrain though, as it is far below the altitude of the flyers.
-* Otherwise, the attack works just like a normal shooting attack -- so for example,as long as you meet the other requirements then you may split fire between multiple target detachments.
+* You can attack only where an enemy unit lies within range *and* inside the forward 90° arc of your Flyer unit.
+Flyer units do not need line of fire, as they can see over all intervening units and terrain.
+* Otherwise, the attack works just like a normal shooting attack -- so for example, as long as you meet the other requirements then you may split fire between multiple target detachments.
 * *Halve the Firepower* of each {transport-x} Flyer unit that will disembark cargo units (and roll to lock on with any super-heavy weapons).
 . After you resolve the shooting effects, disembark the cargo units just like you would with a ground transportation unit.
-But note:
-* In each case, measure from the *centre* of the transport flyer's flight stand.
+//But note:
+//* In each case, measure from the *centre* of the transport flyer's flight stand.
 +
 {blank}[TODO: Are there any awkward edge cases where this 'measure from the centre' restriction poses a problem? If yes, what is the best alternative given the variety of base and model sizes that is even more variable than with ground units?]
-* Give the disembarked detachment whatever orders you prefer, except that you may not choose Overwatch or March orders.
-To be clear, you *may* put the detachment on Assault orders and make assault moves in the usual way this turn.
++
+{blank}[IJW reply: Yes, there are. A 60 mm round base (fairly common for stability purposes) makes it impossible to disembark units on 20 mm square or 25 mm round bases, and even strip bases barely squeeze in by a few mm. There are also 115 x 90 mm oval bases that are sometimes used for SM Landing Craft, or bigger Ork Landa conversions. I suggest using the standard disembarkation rules, and add a NOTE saying that landing Flyers can't be placed within 10 cm of enemy units and are Vehicles for terrain purposes.
++
+Plus there are players that use scenic bases rather than traditional flight stands, so there is no central column to measure from anyway.]
+
+*  You can only give the disembarking detachment normal orders or Assault orders.
```

Comment:
Yes, it's intentional that Special orders are excluded. 

This is partly because the existing Special orders all need to be declared in the Start phase, before the detachment has arrived on the table, and partly for parity with [Drop Pods](https://thehobby.zone/resources/e40k-compendium/Content/Battles/ScenarioSpecialRules/DropPods.htm):

>Units emerging from drop pods are limited to using assault or normal orders.

----

'Advance' orders is something I'd been meaning to bring up when I get around to checking the rest of the existing Remastered text.

As this is a term that didn't exist in E40k, and there is no note about this being an added term, I had assumed that it was copied from EA and hadn't been corrected to 'normal' orders yet. So I used the 'normal orders' term from E40k, with a view to looking at the Orders and conditions page in a future pass.

Reply:
> Yes, it's intentional that Special orders are excluded.
> 
> This is partly because the existing Special orders all need to be declared in the Start phase, before the detachment has arrived on the table, and partly for parity with [Drop Pods](https://thehobby.zone/resources/e40k-compendium/Content/Battles/ScenarioSpecialRules/DropPods.htm):
> 
> > Units emerging from drop pods are limited to using assault or normal orders.

Cool, that all seems good to me.

---

> 'Advance' orders is something I'd been meaning to bring up when I get around to checking the rest of the existing Remastered text.
> 
> As this is a term that didn't exist in E40k, and there is no note about this being an added term, I had assumed that it was copied from EA and hadn't been corrected to 'normal' orders yet. So I used the 'normal orders' term from E40k, with a view to looking at the Orders and conditions page in a future pass.

I labelled and defined Advance orders early on, before we started the 'Epic 40k difference' notes idea — and yes, this was around the same time that I dallied with a couple of other name changes partially inspired by EA. (I think also I discussed it a bit in the FB group at the time, but I can't really remember now.)

My basic idea was that it's weird to label and define all the other orders but then not to do the same for the 'normal/default/standard/whatever' orders\*. Just because they are 'normal/whatever' doesn't mean that we shouldn't clearly label and define them alongside the rest. And as 'normal' is rather generic as a label and thus apt to be ambiguous in some situations, I chose the more precise term 'Advance' (in part because it is also familiar from EA — rather than choosing yet another term).

*\*And let's not forget, GW didn't even define them as orders — rather, '[Special moves](https://thehobby.zone/resources/e40k-compendium/Content/CoreRules/SpecialMoves.htm)' and "instead of making a normal move". References to 'orders' relied on implication and association with the Order dice.*

It all helps with clarity. I think that 'Advance orders' is a better than 'Normal orders' — for one thing, it's inherently less open to ambiguity. But I'm not fixated on it, if you object.

**However** I do think that whatever name we do use:

- It should have a single defined label that we use consistently, of course.
- If this label **is** to be 'normal' orders, then the generic nature of this word gives all the more reason why we should capitalise it (in common with all the other orders) so that it is always crystal clear that we're referring to a specific, defined thing in the rules — not to orders that could be interpreted as 'normal for the current context', but specifically 'Normal orders'.
- We should keep the corresponding clear definition alongside all the other orders. 

In my view, the problem isn't that I've created a new label, but rather that GW didn't actually define a label and meaning properly in the first place. I sought to fix that.